phpBB/phpbb/db/tools/tools.php

```diff
@@ -1413,6 +1413,33 @@ function sql_table_drop($table_name)
 		return $this->_sql_run_sql($statements);
 	}
 
+	/**
+	 * {@inheritDoc}
+	 */
+	function sql_table_truncate($table_name)
+	{
+		$statements = array();
+
+		if (!$this->sql_table_exists($table_name))
+		{
+			return $this->_sql_run_sql($statements);
+		}
+
+		switch ($this->sql_layer)
+		{
+			case 'sqlite':
```

Comment:
I see there is only sqlite3 available here
https://github.com/akbarhashmi/phpbb/blob/5186570bac0af2a96ea8770c4436bb73b35c7763/phpBB/phpbb/db/tools/tools.php#L139

Reply:
Removed case: sqlite 
Hope it passes this time

Reply:
Looks good, the Devs will say though.